plugin/build.gradle

```diff
@@ -184,6 +184,14 @@ integTest {
     // Set this to true this if you want to see the logs in the terminal test output.
     // note: if left false the log output will still show in your IDE
     testLogging.showStandardStreams = true
+
+    // Exclude integration test for search index tool for multiple nodes
+    // because we cannot get concrete exception message in multi-node cluster
```

Comment:
@yuye-aws  Why can't get concrete exception message ?

Reply:
In multi node cluster, the original exception message is wrapped with `RemoteTransportException`, the cause of which is still `RemoteTransportException`.

Reply:
Since this PR is just moving code from skills repo to ml-commons, we can keep it as is. But please create a Github issue for supporting multi-node cluster for this IT.

Reply:
Created a bug issue: https://github.com/opensearch-project/ml-commons/issues/2362